src/nomad_neb_workflows/schema_packages/neb.py

```diff
@@ -61,14 +60,14 @@ def extract_total_energy_differences(
             Optional[pint.Quantity]: The total energy differences of the system for each of the images in the path of configurations in units of energies.
         """
         # Resolve the reference of energies from the first NEB task
-        scf_task = self.tasks[1]
-        if scf_task.m_xpath('outputs[0].section.energy.total.value') is None:
+        ref_image_task = self.tasks[0]
+        if ref_image_task.m_xpath('outputs[0].section.energy.total.value') is None:
             logger.error(
                 'Could not resolved the initial value of the total energy for referencing.'
             )
             return None
-        energy_reference = scf_task.outputs[0].section.energy.total.value.m
-        energy_units = scf_task.outputs[0].section.energy.total.value.u
+        energy_reference = ref_image_task.outputs[0].section.energy.total.value.m
+        energy_units = ref_image_task.outputs[0].section.energy.total.value.u
 
         # Append the energy differences of the images w.r.t. the reference of energies
         tot_energies = []
```

Comment:
I just realized the next check (l.75ff) is too strict: the energy differences could of course also be negative if we have a exothermic reaction. We should change it so that at least one image (the transition state = maximum) should be positive. (Or we remove the check entirely here).

Reply:
Yeah, just remove the check. I can also do it if you want, but I don't know if @u-gajera is working on this branch or what is the plan.

Reply:
Hi @JosePizarro3 , you can remove this as well. I am waiting to finilize this branch and start the plotting again from this point.